substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/SubstrateJVM.java

```diff
@@ -495,6 +495,16 @@ public boolean isEnabled(JfrEvent event) {
         return eventSettings[(int) event.getId()].isEnabled();
     }
 
+    @Uninterruptible(reason = "Called from uninterruptible code.", mayBeInlined = true)
+    public void setLarge(long eventTypeId, boolean large) {
```

Comment:
`setLarge(JfrEvent event, boolean large)` is probably easier to use in the callers.